workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Ipc.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Newtonsoft.Json;
+
+namespace Improbable.Gdk.DeploymentLauncher
+{
+    /// <summary>
+    ///     Methods for inter-process communication with the parent process (Unity). All via JSON.
+    /// </summary>
+    public static class Ipc
+    {
+        public enum ErrorCode : uint
+        {
+            FileNotFound = 1,
```

Comment:
You already have a `NotFound` error below, seems redundant since `NotFound` should make sense in context too

Reply:
Ah right - makes sense 😄